TGM/src/main/java/network/warzone/tgm/command/MiscCommands.java

```diff
@@ -49,8 +50,8 @@ public static void ping(CommandContext cmd, CommandSender sender) {
     @Command(aliases = {"nicks"}, desc = "View all nicked players")
```

Comment:
While this is not part of the PR, please move all the nick commands to a new dedicated class for them as the comment above the class declaration states that the commands must not implement any API functionality.